resolwe_bio/tests/processes/test_upload.py

```diff
@@ -400,3 +400,68 @@ def test_upload_nucl_seq(self):
         seq = self.run_process('upload-fasta-nucl', inputs)
         self.assertFile(seq, 'fasta', 'genome.fasta.gz', compression='gzip')
         self.assertFields(seq, 'species', 'Dictyostelium discoideum')
+
+    @tag_process('basespace-import', 'basespace-fastq-single')
+    def test_basespace_fastq_single(self):
+        with self.preparation_stage():
+            # Token with limited scope pre-obtained from dedicated BaseSpace testing app.
+            handle = Secret.objects.create_secret('9bdf059c759a429f8af52ca084130060', self.admin)
+
+            import_inputs = {'file_ids': ['9461130722', '9461121664'], 'access_token_secret': {'handle': handle}}
+            basespace_import = self.run_process('basespace-import', import_inputs)
+
+        fastq_single_inputs = {'reads': basespace_import.pk}
+        reads = self.run_process('basespace-fastq-single', fastq_single_inputs)
+
+        self.assertFiles(reads, 'fastq', ['Test_S1_L001_R1_001.fastq.gz',
+                                          'Test_S1_L002_R1_001.fastq.gz'], compression='gzip')
+        del reads.output['fastqc_url'][0]['total_size']  # Non-deterministic output.
+        del reads.output['fastqc_url'][1]['total_size']  # Non-deterministic output.
+        self.assertFields(reads, 'fastqc_url', [{'file': 'fastqc/Test_S1_L001_R1_001_fastqc/fastqc_report.html',
+                                                 'refs': ['fastqc/Test_S1_L001_R1_001_fastqc'],
+                                                 'size': 309578},
```

Comment:
If `size` is changed to a random number, the tests still pass. How come?

Reply:
File size is not being checked anymore https://github.com/genialis/resolwe/blob/master/resolwe/test/testcases/process.py#L480, you can remove it.

Reply:
I guess `total_size` still needs to be [removed](https://github.com/genialis/resolwe-bio/pull/479/files#diff-c3decfeee6f2d762429890905d989817R418)?

Reply:
Yes.